tools/pkglint/lib/rules.ts

```diff
@@ -788,6 +788,12 @@ export class NoJsiiDep extends ValidationRule {
   public readonly name = 'dependencies/no-jsii';
 
   public validate(pkg: PackageJson): void {
+    // don't do this check for the separately-vended modules,
+    // which need to depend on jsii-rosetta
+    if (pkg.packageName.startsWith('@aws-cdk-lib-alpha/')) {
+      return;
+    }
```

Comment:
curious: why is this changing for these packages?

Reply:
They need to depend on `jsii-rosetta` in order to execute the `rosetta:extract` task.

Reply:
Why is this special to alpha packages? Isn't this true for all our packages, v1 and v2?

Reply:
The V1 packages get `jsii-rosetta` from the main monorepo. That doesn't work for the inner monorepo that the alpha modules live in (they binary cannot be found), so they need an explicit dependency on that package.

Reply:
I still feel we haven't gotten to the bottom of this. I don't understand why it works for v1 and not for v2 alpha packages.

What exactly about the v1 setup makes it such that the `jsii-rosetta` binary is available? 
Why can't the same be done for v2 alpha packages?

Reply:
I still feel we haven't gotten to the bottom of this. I don't understand why it works for v1 and not for v2 alpha packages.

What exactly about the v1 setup makes it such that the `jsii-rosetta` binary is available? 
Why can't the same be done for v2 alpha packages?

Reply:
Discussed offline. Removed from this PR.